src/tests.rs

```diff
@@ -261,7 +261,11 @@ fn test_region_tag() -> Result<(), Box<dyn std::error::Error>> {
 
     let data = match region_entry.data {
         IndexData::Bin(d) => d,
-        _ => return Err(Box::new(RPMError::new("should be bin"))),
+        _ => {
+            return Err(Box::new(TestError::StringError(
+                "should be bin".to_string(),
```

Comment:
Why bother to return an error here? Can't we simply it with an `expect_matches!()` from crate `expect_matches` and use expects or `assert!(...is_ok())` / `.expect("yada")` otherwise?